src/components/fileviewercov.js

```diff
@@ -1,52 +1,75 @@
-/* This file contains file-view RHS siderbar components */
-
+/* This file contains coverage information for a particular revision of a source file */
 import React from 'react';
 
+const _ = require('lodash');
+
 /* Sidebar component, show which tests will cover the given selected line */
-export const TestsSideViewer = ({testsPerLines, selectedLine}) => {
+export const TestsSideViewer = ({ coverage, lineNumber }) => {
+  let content;
 
-  if (!testsPerLines) {
-    return (
-      <div className="tests_viewer">
-        <h3>"Fetching coverage from backend..."</h3>
-      </div>
-    );
+  if (!coverage) {
+    content = <h3>Fetching coverage from backend...</h3>;
+  } else if (!lineNumber) {
+    // TODO if no line has been selected, show coverage of the file
+    content = <h3>Select a line to view tests</h3>;
+  } else {
+    /* Parse coverage data returned from ActiveData to get the tests for a line */
+    const testThatCover = coverage.data.filter(d => d.source.file.covered.find(line => line === lineNumber));
```

Comment:
I agree with @LinkaiQi, this is a good line!  `includes` is a bit more direct:

    const testThatCover = coverage.data.filter(d => d.source.file.covered.includes(lineNumber));